### ISSUE TITLE
rajawali3d.org

### ISSUE THREAD

**Issue by ToxicBakery** [Rajawali/Rajawali#634]
**DECISIONS WILL BE MADE SOON ON A SITE, MAKE SURE YOU CHECK OUT THE MOCKUPS AS THE WEEK GOES ON AND LEAVE YOUR FEEDBACK [Mockups for new site](https://github.com/MasDennis/Rajawali/wiki/Mockups-for-new-site)**

As you may know we are moving to put the project in a more organized state. Part of this organization effort is to create a website that better supports all of the useful documentation for the Engine. As such we need a plan on what the site should look like and what functions it will serve. 
## Status
- [x] What features must the site have, we need to answer this before we can move forward at all. Requirements requirements requirements!
- [x] How do we edit pages, what limitations are we bound to? I am guessing that the site is limited to HTML only so we can not run any server side goodness.
- [x] How should the site look? This will likely be a process of finding some candidates and slowly narrowing it down to something we can all, or just Dennis, agree on.
## Requirements
- A list of features, particularly those that make Rajawali stand out. For me being 4.0+ and no NDK makes Rajawali immensely attractive now.  
- A general breakdown of how the rendering process in Rajawali is intended to work
- Host the wiki? Or should this stay put?

_Overall_
- A simple, uncluttered layout. Responsive (mobile/desktop friendly).
- Small Rajawali logo (top left)
- Menu at the top (top right) (Gallery, Tutorials, Documentation, Issues, Team)
- Light colours
- Google AdSense integration

_Homepage_
- Short description of the framework with some of the key features (4.0+, no NDK, ES 2.0, etc)
- 6 or 8 square images of the most beautiful Rajawali apps/wallpapers. With a link to the complete gallery.
- "Start building 3D content now" with a link to Github (tags & unstable version w/ disclaimer)
- Icons w/ short description + links to same items as menu (Gallery, Tutorials, Documentation, Issues)

_Gallery_
- Square tiles with links to Google Play. Clean, simple layout

_Documentation_
- JavaDoc but sexier than it currently is, well integrated with the site

_Issues_
- Basically just links to the Github Issues page

_Team_
- Team profiles. Avatar, short description, role + social media links.

Mods, feel free to edit this post as needed to add clarity or update as progress is made.
